### DISCUSSION TITLE
Is the multi currency & multi language feature already available ?

### DISCUSSION THREAD

**Discussion by gelsas** [saleor/saleor#7858] [Q&A] [answered]
Hi,

Is the multi currency & multi language feature already available ? I read that is will be introduced with saleor 3.0.
Is there any page where I can read more how the multi currency works and how to handle multi languages ?

Best,
gelsas

**Comment by karolkielecki** [comment, marked as answer]
@gelsas Hi, 

Saleor 3.0 introduces multichannel w/ multicurrency. Beta is now available and we recommend using it: https://github.com/mirumee/saleor/releases. You can find more info about Saleor 3.0 here: https://docs.saleor.io/docs/3.0/.

Regarding storefront translations--we've added react-intl integration. Check it out: https://github.com/mirumee/saleor-storefront/pull/782

Cheers,
